OpenSim/Tests/README/testREADME.cpp

```diff
@@ -93,22 +93,33 @@ int main() {
         "elbow_angle");
     model.addComponent(reporter);
 
+    // Add display geometry.
+#ifdef VISUALIZE
```

Comment:
I don't think you have to guard against adding the Geometry. If the Model is not visualized the Geometry won't be visible but it will still be part of the Model.